lib/routes/account.js

```diff
@@ -470,19 +470,22 @@ module.exports = function (
         function extractUnblockCode (e) {
           // If it's a customs-related error...
           if (e.errno === error.ERRNO.REQUEST_BLOCKED || e.errno === error.ERRNO.THROTTLED) {
-            request.emitMetricsEvent('account.login.blocked')
-            var verificationMethod = e.output.payload.verificationMethod
-            // ...and if they can verify their way around it,
-            // then extract any unblockCode from the request.
-            // We'll re-throw the error later if the code is invalid.
-            if (verificationMethod === 'email-captcha') {
-              unblockCode = request.payload.unblockCode
-              if (unblockCode) {
-                unblockCode = unblockCode.toUpperCase()
-              }
-              customsErr = e
-              return
-            }
+            return request.emitMetricsEvent('account.login.blocked')
```

Comment:
Just to say explicitly: I think this is the right call, and we can happily pay a little in response latency if it means reliable tests.  We can revisit if metrics one day tell us that the latency of these requests is a bottleneck.